### ISSUE TITLE
Event at Findout Technologies

### ISSUE THREAD

**Issue by pfilipsson** [stockholm-react-js/meetups#18]
# My idea
FindOut Technologies offers their office (Arne Beurlings Torg 9A) for the November event of Stockholm ReactJS Meetup Suggested date: November 20. The topic could be UX for developers in a React context. Another alternative could be something related to Ericsson. They are for instance working on a web based IDE called Theia. FindOut has contact with that team. 

# Help needed
FindOut can take care of seating, food etc. We need help with finding speakers. Depending on the theme we might need help finding speakers.

**Comment by Pallinder** [comment]
Great! This event would take place in Kista, right?

**Comment by pfilipsson** [comment]
Yes, the idea is that the event would take place in Kista, in front of the Ericsson HQ. That is why it could be good to try to get Ericsson people to talk if possible. I guess we need to check that Kista is not a problem for attendees.

**Comment by Pallinder** [comment]
Hi there, should we reschedule this?
